### PR TITLE
Partial refund with specific order items

### PR DESCRIPTION
**Pull request by aait**

### CODE REVIEW

**Review by asbjornu — CHANGES_REQUESTED** [review]
Can we have a test that exercises the added code, please?

File: tests/unit-tests/wc-gateway-swedbank-pay-checkout.php [hunk 1]
```diff
@@ -108,4 +108,26 @@ public function test_process_refund() {
...
+			$args['line_items'][$order_item->get_id()] = array(
+				'qty' => 1,
+				'refund_total' => $order_item->get_total(),
+				'refund_tax' => 0,
+			);
+		}
+
+		WC()->session->set( 'swedbank_refund_parameters', $args );
+		$result = $this->gateway->process_refund( $order->get_id(), 1, 'Test' );
+		$this->assertInstanceOf( 'WP_Error', $result );
```
Comment (asbjornu) [inline tests/unit-tests/wc-gateway-swedbank-pay-checkout.php:131]: Is it not possible to perform a test that doesn't end with an error?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: tests/unit-tests/wc-gateway-swedbank-pay-checkout.php (hunk 1 above)
Comment (aait) [reply tests/unit-tests/wc-gateway-swedbank-pay-checkout.php:131]: We should make payment to test refund in this case. Not sure that WooCommerce provide such test tools like that

Comment (asbjornu) [reply tests/unit-tests/wc-gateway-swedbank-pay-checkout.php:131]: You probably need to write some UI tests with Selenium or something similar to go through the entire purchase process.

Comment (aait) [reply tests/unit-tests/wc-gateway-swedbank-pay-checkout.php:131]: @asbjornu Sorry, we have a lot priority tasks at the momement. Can we implement UI tests later please?

Comment (asbjornu) [reply tests/unit-tests/wc-gateway-swedbank-pay-checkout.php:131]: Yes, please add a task for it in Jira.

**Review by asbjornu — APPROVED** [review]

**Pull request merged**